File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        int sum =0;
        int multi = 1;
        
        for(int i = 0; i<num_list.length; i++){
            if(num_list.length>=11){
                sum += num_list[i];
                answer = sum;}
            else{
                multi *= num_list[i];
              answer = multi;}
            
        }
        return answer;
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
   let answer = 0;
    let answer1 = 1;
    for (let i = 0; i < num_list.length; i++) {
        if (num_list.length >= 11) {
            answer += num_list[i];
        } else {
            answer1 *= num_list[i];
        }
    }
    return num_list.length >= 11 ? answer : answer1;
}